src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseLookupService.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.ecm.common.client.CcdClient;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.SubmitEvent;
+import uk.gov.hmcts.et.common.model.multiples.MultipleData;
+import uk.gov.hmcts.et.common.model.multiples.MultipleDetails;
+
+import java.io.IOException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class CaseLookupService {
+    public static final String EMPLOYMENT = "EMPLOYMENT";
```

Comment:
Is it worth putting this in a Constants class? I'm pretty sure I've seen a lot of declarations for EMPLOYMENT and it makes sense for it to be a constant